paddle/fluid/pir/dialect/operator/ir/ops.yaml

```
@@ -469,6 +469,14 @@
   optional : in_accum, in_state, out_scale, out_accum, out_state
   inplace : (scale -> out_scale, in_accum -> out_accum, in_state -> out_state)
 
+- op : dgc_momentum
+  args : (Tensor param, Tensor grad, Tensor velocity, Tensor learning_rate, Tensor master_param, Tensor current_step, Tensor nranks, float mu, bool use_nesterov=false, str regularization_method="", float regularization_coeff=0.0f, bool multi_precision=false, float rescale_grad=1.0f, float rampup_begin_step=-1.0f)
+  output : Tensor(param_out), Tensor(velocity_out), Tensor(master_param_out), Tensor(grad_out)
+  kernel :
+    func : dgc_momentum
```

```suggestion
    func : dgc_momentum
    param: [和kernel一致的参数顺序]
```
args的参数顺序是可以调整的，有时也是必要的。但是这个时候 kernel和infermeta中就需要指定一个和函数调用所需参数一致的顺序，可以参考 #61293 ，也遇到了类似问题。

---

多谢，但我这里好像是参数后面写错了，后面顺序是一致的